octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py

```diff
@@ -799,26 +800,17 @@ async def order(self, order: cryptofeed_types.OrderInfo, receipt_timestamp: floa
         :param order the order object defined in cryptofeed.types.?
         :param receipt_timestamp: received timestamp
         """
-        await self.push_to_channel(trading_constants.ORDERS_CHANNEL, [{
-            trading_enums.ExchangeConstantsOrderColumns.TYPE.value: self._parse_order_type(order.type),
-            trading_enums.ExchangeConstantsOrderColumns.STATUS.value: self._parse_order_status(order.status),
-            trading_enums.ExchangeConstantsOrderColumns.ID.value: order.id,
-            trading_enums.ExchangeConstantsOrderColumns.SYMBOL.value: order.symbol,
-            trading_enums.ExchangeConstantsOrderColumns.PRICE.value: order.price,
-            trading_enums.ExchangeConstantsOrderColumns.AMOUNT.value: order.amount,
-            trading_enums.ExchangeConstantsOrderColumns.REMAINING.value: order.remaining,
-            trading_enums.ExchangeConstantsOrderColumns.FILLED.value: order.amount - order.remaining,
-            trading_enums.ExchangeConstantsOrderColumns.TIMESTAMP.value: order.timestamp,
-            trading_enums.ExchangeConstantsOrderColumns.SIDE.value: self._parse_order_side(order.side)
-        }])
+        order_parser = OrdersParser(self.exchange)
```

Comment:
Do we need a cryptofeed order parser or it's already working with the current order parser?

Reply:
I havent tested it yet, but I included all logic, so it should work

Reply:
I don't think this work. For example how do you handle edge cryptofeed status parsed in the old `_parse_order_status`?
![image](https://user-images.githubusercontent.com/10454670/205758746-32fb55f4-7b26-4ec6-8307-29446429f012.png)

Reply:
there is now a dedicated cryptofeed parser